### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by reggie7**

### CODE REVIEW

File: platform/base/core/src/main/java/com/peregrine/adaption/impl/PeregrineAdapterFactory.java [hunk 1]
```diff
@@ -38,11 +38,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.jcr.Node;
-import javax.jcr.RepositoryException;
-
 import static com.peregrine.commons.util.PerConstants.*;
-import static com.peregrine.commons.util.PerConstants.PER_REPLICATION;
```
Comment (reusr1) [inline platform/base/core/src/main/java/com/peregrine/adaption/impl/PeregrineAdapterFactory.java:45]: we probably should remove the line before instead

Comment (reggie7) [reply platform/base/core/src/main/java/com/peregrine/adaption/impl/PeregrineAdapterFactory.java:45]: Hmm, good catch, though I suggest this instead: let's keep the refactoring PRs a single-topic ones. Maybe I deal with `.*` imports in a separate one? I hear that the team prefers them expanded, so we can expand them all in another PR. Can we do it this way?

**Review by flx-sta — APPROVED** [review]
i dont know everything of the backend but looks fine for me
image
image

**Review by cmrockwell — APPROVED** [review]
Looks okay to me

**Pull request merged**